nowcasting_dataset/manager.py

```diff
@@ -316,6 +322,81 @@ def _find_splits_which_need_more_batches(
                 splits_which_need_more_batches.append(split_name)
         return splits_which_need_more_batches
 
+    def create_derived_batches(self, overwrite_batches: bool) -> None:
```

Comment:
this is duplcate of batches, isnit, is there perhaps a better way to do it, the create batches has an option to do 'data_sources' or 'derrived_data_sources'